packages/devtools-reps/src/object-inspector/tests/component/window.js

```diff
@@ -0,0 +1,102 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const { mount } = require("enzyme");
+const React = require("react");
+const { createFactory } = React;
+const ObjectInspector = createFactory(require("../../index"));
+const {
+  createNode,
+} = require("../../utils");
+
+const gripWindowStubs = require("../../../reps/stubs/window");
+
+describe("ObjectInspector - dimTopLevelWindow", () => {
+  it("renders collapsed top-level window when dimTopLevelWindow is true", () => {
+    const windowNode = createNode(
+      null,
+      "window",
+      "windowpath",
+      {value: gripWindowStubs.get("Window")}
+    );
+
+    // The window node should have the "lessen" class when collapsed.
+    let oi = mount(ObjectInspector({
+      autoExpandDepth: 0,
+      roots: [windowNode],
+      dimTopLevelWindow: true,
+      getObjectProperties: () => {},
+      loadObjectProperties: () => {},
+    }));
+    expect(oi.find(".node.lessen").exists()).toBeTruthy();
```

Comment:
I think these tests could benefit from a defaults/overrides pattern that we use in our component tests:

https://github.com/zaggy/debugger.html/blob/update-webpack/src/components/SecondaryPanes/tests/Expressions.js#L6-L33

It dries up the tests, which makes future changes smaller as default props are defined in one place